handbook/product.md

```diff
@@ -319,6 +319,8 @@ We have certain design conventions that we include in Fleet. We will document mo
 
 Use `---`, with color `$ui-fleet-black-50` as the default UI for empty columns.
 
```

Comment:
```suggestion

**Form behavior**

```

Reply:
It looks like we're currently using a "Table empty states" header above the "Use `---`..." design convention.

I suggested adding a similar "Form behavior" header here, however we could also just remove the "Table empty states" header because the list is quite short and easily scannable now. Your call :)